src/core/thread/mle.cpp

```diff
@@ -2139,10 +2139,6 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
             mNetif.GetPendingDataset().Set(pendingTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
-    else
```

Comment:
Above, there is logic to check if the Active Timestamp is `0` or not and either clear or set the Active Dataset appropriately.  If the Active Timestamp and Active Operational Dataset TLVs are included, should we always set the Active Dataset as well?